### ISSUE TITLE
Is this usable for my algo trading bot?

### ISSUE THREAD

**Issue by abhyuditjain** [zerodhatech/kiteconnect-rs#38]
Hi,

I wanted to make a bot in Rust for my algo trading. Is this crate usable? Are there any APIs which are missing/obsolete?

I see that there are no commits in a while and there are some open issues.

Thanks

**Comment by joeirimpan** [comment]
Hi @abhyuditjain, few apis pertaining to gtt, margin calculation are missing.

**Comment by abhyuditjain** [comment]
@joeirimpan Thanks for responding. Any plan on adding those?

**Comment by skittishdev** [comment]
Is this library still maintained/officially supported? It's been a couple of years since there were any commits.

**Comment by ranjanrak** [comment]
Yes, the rust client is no longer actively maintained. However, apart from a few endpoints that were introduced in the last couple of years, the project is still in a working state. We have decided to archive it now so that users can have a clear understanding. We plan to re-work on adding those missing endpoints in the future.
